src/main/java/au/id/micolous/metrodroid/fragment/CardsFragment.java

```diff
@@ -120,26 +151,27 @@ public void onLoaderReset(Loader<Cursor> cursorLoader) {
     public void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setHasOptionsMenu(true);
-
-        mDataCache = new HashMap<>();
     }
 
     @Override
     public void onViewCreated(View view, Bundle savedInstanceState) {
         super.onViewCreated(view, savedInstanceState);
 
         registerForContextMenu(getListView());
```

Comment:
This currently raises a NullPointerException when the view is created.  Removing it makes it not crash, but it would be nice to be able to have the same functionality as before (previously you could delete scans from here, so would be good to be able to delete all scans of a card, or single scans here).

Reply:
I lost a hunk when splitting. Fixed.